pkg/raw/raw_exec.go

```diff
@@ -5,44 +5,101 @@ package raw
 
 import (
 	"fmt"
-	"os/exec"
-	"strconv"
+	"io"
+	"os"
 
+	"github.com/go-logr/logr"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
 )
 
-type Exec struct {
-}
+type Exec struct{}
+
+const filePerm = 0644
```

Comment:
why 0644? i think 0660 or 0640 will be better. Little depends how vm will access to this file. So i mean libvirt-qemu user has to has access

Reply:
changed it to 0640

Reply:
are you sure?

Reply:
The raw file is getting created with the user with which the controller is run. So, it looks like working.

Reply:
yes and this file has to be attach to vm, which need to do operation read and write. So if controller doesn't use libvirt-qemu user, probably you will end with permission error.

Reply:
As discussed offline, we have decided to provide users with more flexibility, allowing them to use different groups and users for each component. Therefore, we can set the permission to 0660 here.